src/components/NewTodoForm/NewTodoForm.js

```diff
@@ -5,19 +5,19 @@ import './NewTodoForm.css';
 
 export class NewTodoForm extends React.Component {
   state={
-    username: 'Choose a user',
+    username: '',
     title: '',
-    errortitle: false,
-    errorusername: false,
+    titleError: false,
+    usernameError: false,
   }
 
   handleChange = (event) => {
     const { name, value } = event.target;
-    const re = /[^\w ]+/;
+    const regExp = /[^\w ]+/;
 
     this.setState({
-      [name]: value.replace(re, ''),
-      [`error${name}`]: false,
+      [name]: value.replace(regExp, ''),
+      [`${name}Error`]: false,
```

Comment:
let's create an 'errors' field in the state and put there all the errors info, so then you can change each error value like this
`erros[errorName]: false`

Reply:
`erros[errorName]: false` didn't work as it should or maybe I did something wrong.

Reply:
I made it working only in such a variant```
this.setState(state => ({
      [name]: value.replace(regExp, ''),
      errors: {
        ...state.errors,
        [name]: false,
      },
    }));
```